docs/user/03-10-preconfigured-secret.md

```diff
@@ -29,7 +29,7 @@ The `sap-btp-manager` Secret provides the following credentials:
 
 > [!NOTE]
 > If you modify or delete the `sap-btp-manager` Secret, it is modified back to its previous settings or regenerated within up to 24 hours.
-> However, if the Secret is labeled with `kyma-project.io/skip-reconciliation: "true"`, the job skips the reconciliation for this Secret.
+> To stop the reconciliation, label the Secret with `kyma-project.io/skip-reconciliation: "true"`. For more information, see [Customize Credentials and Access](03-11-customize_secret.md).
```

Comment:
There is implicit assumption that reader is aware that `reconciliation` == `being modified back or regenerated`.
It was already in the previous version, but still.